static/docs/reference/generated/kubectl/node_modules/jquery/dist/jquery.min.js

```diff
@@ -1,2 +1,2 @@
-/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.org/license */
```

Comment:
In my generating, the version of JQuery changes from v3.6.0 to v3.6.1, but in your generating, it is not the same. I'm curious about this change. My generating is as below.
![image](https://user-images.githubusercontent.com/42922129/210481218-2f919534-a351-4af4-90d7-f64fab4a2cc4.png)

Reply:
I'm not sure about the reason why this thing gets a version bump.